e2e-tests/portal/cypress/e2e/journeys/admin/create-user/update-a-user.spec.js

```diff
@@ -139,7 +142,24 @@ context('Admin user updates an existing user', () => {
     editUser.save().click();
 
     // Assert role input error
-    createUser.rolesError().should('contain', 'The admin role can only be associated with a UKEF email address');
+    createUser.rolesError().should('contain', 'Admin and read-only roles can only be associated with a UKEF email address');
```

Comment:
Hi @rkontogianni  sorry for the confusion, read-only roles can be assigned to non UKEF users to confirm:

* Admin - UKEF only
* Read-only - Any user
* All bank - UKEF only
* All bank - Admin and read-only role types only